From 2c001c4912256f68761c909b9e01be61413397b0 Mon Sep 17 00:00:00 2001
From: rany <ranygh@riseup.net>
Date: Tue, 27 Aug 2024 22:29:08 +0530
Subject: [PATCH 2/2] config: common: Disable blur on app launch

This change adds the system property that disables blur during app
launch.

Change-Id: I4457ad4757ab656435d8454a4f8d15aa136db857
---
 config/common.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/config/common.mk b/config/common.mk
index 7e367702..47b3c34b 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -181,6 +181,9 @@ ifeq ($(TARGET_ENABLE_BLUR),true)
     USES_BLUR=1
 else
     USES_BLUR=0
+
+PRODUCT_PRODUCT_PROPERTIES += \
+    ro.launcher.blur.appLaunch=0
 endif
 
 PRODUCT_PRODUCT_PROPERTIES += \
-- 
2.39.5

